dist/auth.js

```diff
@@ -15,12 +15,13 @@
  * @packageDocumentation
  */
 import { diagnostic } from './utils/logger.js';
-const API_BEARER_TOKEN = process.env.API_BEARER_TOKEN;
```

Comment:
I wonder if we should remove `dist` from our source history and only commit the `src`? It would make it easier to review and we can always generate the `dist` from the `src`. What do you think @KagemniKarimu?